nova_vm/src/ecmascript/types/language/value.rs

```diff
@@ -443,6 +443,14 @@ impl Value {
         to_uint32(agent, self)
     }
 
+    pub fn to_int16(self, agent: &mut Agent) -> JsResult<i16> {
```

Comment:
**issue(nonblocking):** I don't think these functions are really worth exposing through the `Value`'s public API, or on the `Value` API at all.

The spec defines `ToUint16` as an abstract operation that takes a `Value`, not as a method of `Value`. Hence, I'd prefer matching spec call logic so when the spec says `ToUint16(value)` we do `to_uint16(agent, value)`

Reply:
Would this also then apply to `to_uint32` and `to_int32`?

Reply:
Yeah.